src/databricks/labs/ucx/source_code/directfs_access.py

```diff
@@ -50,7 +50,7 @@ def dump_all(self, dfsas: Sequence[DirectFsAccess]) -> None:
         Providing a multi-entity crawler is out-of-scope of this PR
         """
         try:
-            # TODO until we historize data, we append all DFSAs
+            # TODO: @JCZuurmond until we historize data, we append all DFSAs
```

Comment:
@asnare : Do you recall what we should do here? As we now are historizing the DFSAs, still we need to get the snapshot from the inventory

Reply:
The intent here is that a crawl should override the existing data. In this case we were instead appending so that history would be kept, pending implementation of the history log.

So this can be updated to overwrite instead of append, with the major proviso that we check to make sure the data is being written once for a crawl. (For a while a single "crawl" was writing its data across several operations, which won't work as intended if the mode for such operations is changed to `overwrite`.)

Reply:
I decide to not update the mode to "overwrite" yet as we do not historise the DFSA from QueryLinter (yet). 

I did confirm there is one single "crawl" per run - of `WorkflowLinter` and `QueryLinter` separately (different DFSAs).